tasks/deploy.js

```diff
@@ -117,8 +118,17 @@ module.exports = function(options) {
 		if (fastDeployPaths.tempDest) {
 			stream.pipe(gulp.dest(fastDeployPaths.tempDest));
 		}
-
-		stream.pipe(livereload());
+		
+		stream.pipe(es.map(function(file, done) {
```

Comment:
Hey @izaera,

Do you think https://www.npmjs.com/package/gulp-util#buffercb will work for this case? Just asking in case we can avoid adding another dependency.